lua/mLRS.lua

```diff
@@ -1153,24 +1163,46 @@ local function drawPageMain()
     end
 
     if DEVICE_PARAM_LIST_complete and DEVICE_PARAM_LIST[3].allowed_mask > 0 then
-        lcd.drawText(270, y, "Ortho", g_textColor)
-        local p = DEVICE_PARAM_LIST[3] -- param_idx = 3 = RfOrtho
-        if p.options[p.value+1] ~= nil then
-            lcd.drawText(330, y, p.options[p.value+1], cur_attr_p(RFOrtho_idx,3))
+        if LCD_W == 320 then
+             lcd.drawText(LCD_W_HALF+30, y + 42, "Ortho", g_textColor)
+             local p = DEVICE_PARAM_LIST[3] -- param_idx = 3 = RfOrtho
+             if p.options[p.value+1] ~= nil then
+                 lcd.drawText(LCD_W_HALF+90, y + 42, p.options[p.value+1], cur_attr_p(RFOrtho_idx,3))
+             end
+        else
+            lcd.drawText(LCD_W_HALF+30, y, "Ortho", g_textColor)
+            local p = DEVICE_PARAM_LIST[3] -- param_idx = 3 = RfOrtho
+            if p.options[p.value+1] ~= nil then
+                lcd.drawText(LCD_W_HALF+90, y, p.options[p.value+1], cur_attr_p(RFOrtho_idx,3))
+            end
         end
     end
 
     y = 171 --166
-    lcd.drawText(10, y, "Edit Tx", cur_attr(EditTx_idx))
-    if not connected then
-        lcd.drawText(10 + 80, y, "Edit Rx", g_textDisableColor)
+    if LCD_W == 320 then
+        y = 158
+        lcd.drawText(10, y, "Tx", cur_attr(EditTx_idx))
```

Comment:
is it really needed to cut it down to Tx and Rx? From the screen shot it seems also Edit Tx and Edit Rx could be made to fit
this potentially would give us better changes to come up with a generic list of values for the position of the elements